Server/Server.cpp

```diff
@@ -71,6 +75,7 @@ void Server::startHubLoop()
 {
     CommunicatorMessage connectionOperation;
 
+    std::signal(SIGSEGV, signalCallbackHandler);
```

Comment:
👀 👀 👀 👀 👀 👀 👀

Reply:
Does it work on Windows ?

Reply:
Sure, it's used like others signal (SIGINT) which are used in another place in the code. Windows will just never use signal.